rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -212,6 +213,15 @@ bool debugger_frame::eventFilter(QObject* object, QEvent* event)
 	{
 		keyPressEvent(static_cast<QKeyEvent*>(event));
 	}
+
+	if (object == m_choice_units && event->type() == QEvent::FocusOut)
+	{
+		if (auto index = m_choice_units->currentIndex(); index >= 0)
+		{
+			m_choice_units->setCurrentText(m_choice_units->itemText(index));
+		}
+	}
```

Comment:
missing comment/explanation why this is needed.

Reply:
Added.

Reply:
isn't that the same as m_choice_units->clearEditText();
?

Reply:
That just makes the textbox empty with placeholder text.